flask_appbuilder/tests/test_api.py

```diff
@@ -1067,6 +1104,29 @@ def test_get_list_base_order(self):
             },
         )
 
+    def test_get_list_custom_base_query(self):
+        """
+            REST Api: Test get list with a base query
+            defined on the API for the list endpoint
+        """
+
+        client = self.app.test_client()
+        token = self.login(client, USERNAME_ADMIN, PASSWORD_ADMIN)
+
+        # test string order asc
+        rv = self.auth_client_get(client, token, "api/v1/model1apibasequery/")
```

Comment:
We should make sure that the filters,order by and pagination still work has expected

Reply:
I'll add some tests!